config/main.py

```diff
@@ -1375,7 +1375,7 @@ def add_portchannel(ctx, portchannel_name, min_links, fallback):
         ctx.fail("{} already exists!".format(portchannel_name))
 
     fvs = {'admin_status': 'up',
-           'mtu': '9100'}
+           'mtu': '9100', 'lacp_key': 'auto'}
```

Comment:
can you add unit test for this?

Reply:
@lguohan  what tests do you believe should be for this? it only adding another key to the DB.
There are already tests on the sonic-swss PR that check that adding this key will behave as expected.
What should I tests in the sonic-utilities side?

Reply:
Expand the existing test to check 'lacp_key' exists after newly generated or version upgraded?